packages/api-server/src/cache/data.ts

```diff
@@ -281,12 +283,14 @@ export function errorResult(reason: string) {
 }
 
 export function parseExecuteResult(res: string) {
-  const jsonRes: PublishExecuteResult = JSON.parse(res);
-  if (jsonRes.status === PublishExecuteResultStatus.Success) {
-    return jsonRes.data!;
+  const executionResult = JSON.parse(res);
+  if (executionResult?.status === PublishExecuteResultStatus.Success) {
+    return (executionResult as PublishExecuteResult).data;
+  } else if (executionResult?.err != null) {
```

Comment:
```suggestion
  }
  if (executionResult?.err != null) {
```
no need to use `else`?